pkg/apis/core/v1beta1/types.go

```diff
@@ -152,14 +152,33 @@ type DataVolumeSourceS3 struct {
 
 // DataVolumeSourceRegistry provides the parameters to create a Data Volume from an registry source
 type DataVolumeSourceRegistry struct {
-	//URL is the url of the Docker registry source
+	//URL is the url of the registry source (starting with docker:// for Docker, or is:// for ImageStream)
```

Comment:
Actually we also accept oci-archive https://github.com/kubevirt/containerized-data-importer/blob/main/pkg/importer/transport.go#L88-L92

Reply:
Fixed